mmedit/datasets/pipelines/augmentation.py

```diff
@@ -31,7 +34,7 @@ class Resize:
 
     Args:
         keys (list[str]): The images to be resized.
-        scale (float | Tuple[int]): If scale is Tuple(int), target spatial
+        scale (float | tuple[int]): If scale is Tuple(int), target spatial
```

Comment:
We can use `(..)` following tuple. For example, `tuple(int)`

Reply:
I simply followed the convention of other functions in augmentation.py, which seems to follow the square brackets convention.